config/src/config/storage_config.rs

```diff
@@ -97,8 +97,8 @@ impl Default for StorageConfig {
             // conservatively safe minimal prune window. It'll take a few Gigabytes of disk space
             // depending on the size of an average account blob.
             storage_pruner_config: StoragePrunerConfig {
-                state_store_prune_window: Some(1_000_000),
-                ledger_prune_window: Some(10_000_000),
+                state_store_prune_window: Some(100_000),
+                ledger_prune_window: Some(100_000),
```

Comment:
I don't think we need to shrink the windows. (the window for state had been 1M before the ledger pruner work, right? And I think it makes sense to keep a fairly large ledger history window)

Reply:
ugh, you haven't sent it for review. Sorry.